styles/colors.ts

```diff
@@ -0,0 +1 @@
+export const COLORS = { main: '#D44645', sub_green: '#A1C298', sub_yellow: '#FBF2CF' };
```

Comment:
const assertion 달아두면 좋을 것 같아요.

```suggestion
export const COLORS = { main: '#D44645', sub_green: '#A1C298', sub_yellow: '#FBF2CF' } as const;
```

Reply:
넵 리뷰 감사합니다 멘토님 ~~!